app/models/customization_template.rb

```diff
@@ -5,11 +5,15 @@ class CustomizationTemplate < ApplicationRecord
   has_many   :pxe_images, :through => :pxe_image_type
 
   validates :pxe_image_type, :presence => true, :unless => :system?
-  validates :name,           :unique_within_region => true
+  validates :name, :uniqueness => {:scope => :pxe_image_type }, if: :not_seeding?
 
   scope :with_pxe_image_type_id, ->(pxe_image_type_id) { where(:pxe_image_type_id => pxe_image_type_id) }
   scope :with_system,            ->(bool = true)       { where(:system => bool) }
 
+  def not_seeding?
+    !system? || pxe_image_type.present?
```

Comment:
Checking both system and presence is overkill since we have a pxe_image_type presence validation unless system? above...  I guess it could just check `!system`